Microsoft.DotNet.Interactive/KernelExtensionLoader.cs

```diff
@@ -17,7 +17,7 @@ public class KernelExtensionLoader
             FileInfo assemblyFile, 
             IKernel kernel, 
             KernelInvocationContext context,
-            IEnumerable<FileInfo> additionalDependencies = null)
+            IEnumerable<string> additionalDependencies = null)
```

Comment:
I think `additionalDependencies` should be able to be removed in favor of something easier to understand. Not sure if you've seen an opportunity to do that but don't be worried about breaking existing extension functionality because it needs to be redone anyway.